apps/content-publishing-worker/src/request_processor/dsnp.announcement.processor.spec.ts

```diff
@@ -7,50 +7,54 @@ import { IRequestJob } from '#types/interfaces/content-publishing';
 import { AnnouncementTypeName, TagTypeEnum } from '#types/enums';
 import { IIpfsConfig, IpfsService } from '#storage';
 import ipfsConfig from '#storage/ipfs/ipfs.config';
-import Redis from 'ioredis-mock';
-import { getRedisToken } from '@songkeys/nestjs-redis';
+import { GenerateMockConfigProvider, GenerateMockProvider, mockRedisProvider } from '#testlib';
+import { LoggerModule } from 'nestjs-pino';
+import { getPinoHttpOptions } from '#logger-lib';
 
 const mockQueue = {
   add: jest.fn(),
 };
 
 // Mock config values
-const mockIpfsConfig: IIpfsConfig = {
+const mockIpfsConfigProvider = GenerateMockConfigProvider<IIpfsConfig>(ipfsConfig.KEY, {
```

Comment:
This is a lot of extra change that isn't really related to this PR, which has 72 files.